xaynetwork/xayn_discovery_app
# TB-3993: Batch sources management updates

### What 🕵️ 🔍

- Uses the newly added `overrideSources` on the engine, instead of updating sources one-by-one

----------

### How to test (please adjust template) 🥼 🔬
- Verify that source management via settings still works
it's still not super fast, due to the long wait from the engine, but better than before when changing many sources

----------

### References 📝 🔗

- [TB-3993](https://xainag.atlassian.net/browse/TB-3993)

----------
